source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,153 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v6.0.
 
 Latest Mattermost Releases:
+- [Release v7.6 - Feature Release](#release-v7-6-feature-release)
 - [Release v7.5 - Feature Release](#release-v7-5-feature-release)
 - [Release v7.4 - Feature Release](#release-v7-4-feature-release)
 - [Release v7.3 - Feature Release](#release-v7-3-feature-release)
-- [Release v7.2 - Feature Release](#release-v7-2-feature-release)
 - [Release v7.1 - Extended Support Release](#release-v7-1-extended-support-release)
 
+## Release v7.6 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release Day: December 15, 2022**
+
+### Important Upgrade Notes
+
+ - Plugins with a webapp component may need to be updated to work with Mattermost v7.6 release and the updated ``React v17`` dependency. This is to avoid plugins crashing with an error about ``findDOMNode`` being called on an unmounted component. While our `starter template <https://github.com/mattermost/mattermost-plugin-starter-template>`_ depended on an external version of ``React``, it did not do the same for ``ReactDOM``. Plugins need to update their ``webpack.config.js`` directives to externalize ``ReactDOM``. For reference, see https://github.com/mattermost/mattermost-plugin-playbooks/pull/1489. Server-side only plugins are unaffected. This change can be done for existing plugins any time prior to upgrading to Mattermost v7.6 and is backwards compatible with older versions of Mattermost.
```

Comment:
Not sure if this is something we've done elsewhere in the changelogs, but it might be nice if everything after the first sentence of this one was on a separate bullet point indented from the first since that first sentence is the important part and the rest is more details about it.

Along those same lines, it'd be nice to add a note about `ExperimentalSettings.PatchPluginsReactDOM` being available to use if they have a plugin that they can't upgrade for whatever reason.